imperatives/imperatives.go

```diff
@@ -507,20 +510,27 @@ func readAddRouteGrafanaNet(s *toki.Scanner, table table.Interface) error {
 	}
 	schemasFile := string(t.Value)
 
-	t = s.Next()
-	if t.Token != word {
-		return errFmtAddRouteGrafanaNet
-	}
-	aggregationFile := string(t.Value)
-	t = s.Next()
-
-	cfg, err := route.NewGrafanaNetConfig(addr, apiKey, schemasFile, aggregationFile)
+	cfg, err := route.NewGrafanaNetConfig(addr, apiKey, schemasFile)
 	if err != nil {
 		return errFmtAddRouteGrafanaNet
 	}
 
+	t = s.Next()
+
 	for ; t.Token != toki.EOF; t = s.Next() {
 		switch t.Token {
+		case optAggregationFile:
+			t = s.Next()
```

Comment:
Why do you need to do this here?

Reply:
The value of `t.Token` when we enter the for loop will be the name of the optional setting (in this case `aggregationFile=`). We need to move to the next token to get the value of the setting.